ui/app/AppLayouts/Chat/views/ContactsColumnView.qml

```diff
@@ -86,34 +86,18 @@ Item {
 
         StatusIconTabButton {
             icon.name: "public-chat"
-            checked: publicChatCommunityContextMenu.visible
-            highlighted: publicChatCommunityContextMenu.visible
-            onClicked: { publicChatCommunityContextMenu.popup(); }
-            StatusPopupMenu {
-                 id: publicChatCommunityContextMenu
-                 closePolicy: Popup.CloseOnReleaseOutsideParent | Popup.CloseOnEscape
-                 StatusMenuItem {
-                     //% "Join public chat"
-                     text: qsTrId("new-public-group-chat")
-                     icon.name: "public-chat"
-                     onTriggered: Global.openPopup(publicChatPopupComponent)
-                 }
-
-                 StatusMenuItem {
-                     //% "Communities"
-                     text: qsTrId("communities")
-                     icon.name: "communities"
-                     onTriggered: Global.openPopup(communitiesPopupComponent)
-                 }
+            onClicked: {
+                Global.openPopup(publicChatPopupComponent)
+                highlighted = false
+                checked = false
```

Comment:
Since we don't use those properties anymore, maybe we could use another component than `StatusIconTabButton`? I think we have `StatusRoundButton` that is simpler and does the job.